### PR TITLE
telezombie redundancy reduction

### PR DESCRIPTION
**Pull request by Sergei-Rudenkov**

### CODE REVIEW

File: telezombie/starter.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+from tornado import gen, ioloop
+from telezombie import api
+
+
+class KelThuzad(api.TeleLich):
+
+    def __init__(self, api_token):
+        super(KelThuzad, self).__init__(api_token)
```
Comment (yarmash) [inline telezombie/starter.py]: Could be:  `super().__init__(api_token)`

File: telezombie/telezombie/api.py [hunk 2]
```diff
@@ -0,0 +1,184 @@
+from __future__ import unicode_literals
```
Comment (yarmash) [inline telezombie/telezombie/api.py]: I believe this is redundant for python3.

**Comment by yarmash** [comment]
There're some style issues: 'Line too long', 'Unused variable', etc. Use `pylint` to clean 'em up.

File: bot_telegram/stackoverflow_rest/so_endpoints.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import tornado.web
```
Comment (Sergei-Rudenkov) [inline bot_telegram/stackoverflow_rest/so_endpoints.py:1]: selection_047

File: bot_telegram/stackoverflow_rest/so_endpoints.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+from tornado import gen, httpclient
+import json
+
+
+class StackOverflowHandler(tornado.web.RequestHandler):
+    @gen.coroutine
+    def get(self, look_up_pattern):
+        url = "https://api.stackexchange.com/2.2/search?order=desc&sort=votes&intitle=%s&site=stackoverflow&page=1&pagesize=1"
```
Comment (yarmash) [inline bot_telegram/stackoverflow_rest/so_endpoints.py]: You could split a long line in Python like this:

```
line = ("foo"
        "bar"
        "baz")
```

Also, you could (and probably should) use [`tornado.httputil.url_concat`](http://www.tornadoweb.org/en/stable/httputil.html#tornado.httputil.url_concat) for constructing urls.

File: bot_telegram/starter.py [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    def __init__(self, api_token, ws_url):
+        super().__init__(api_token)
+        self.ws_url = ws_url
+
+    @gen.coroutine
+    def on_text(self, message):
+        id_ = message.message_id
+        chat = message.chat
+        text = message.text.strip().replace(" ", "-").replace("[()&?]", "")
+        if not self.is_ascii(text):
```
Comment (yarmash) [inline bot_telegram/starter.py:23]: Why not just search for the text as is and print the "No results" message ? Are there any restrictions on the search text?

**Pull request merged**